Gallery pick crash: "Required view 'navigation_view' (snip) was not found"

In my environment (API 24, emulator), choosing an image from the gallery via the app screen results in a crash. It looks like this didn't happen before #752 ae789308088ab67554a306a4af9daba9805f1ee8. (pinging @veyndan)
 
1. Start the app
2. Press the gallery button
3. Pick an image
4. Crash (see below for the log)

```
 E/AndroidRuntime: FATAL EXCEPTION: main
               Process: fr.free.nrw.commons, PID: 5386
               java.lang.RuntimeException: Unable to start activity ComponentInfo{fr.free.nrw.commons/fr.free.nrw.commons.upload.ShareActivity}: java.lang.IllegalStateException: Required view 'navigation_view' with ID 2131689604 for field 'navigationView' was not found. If this view is optional add '@Nullable' (fields) or '@Optional' (methods) annotation.
                   at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2646)
                   at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:2707)
                   at android.app.ActivityThread.-wrap12(ActivityThread.java)
                   at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1460)
                   at android.os.Handler.dispatchMessage(Handler.java:102)
                   at android.os.Looper.loop(Looper.java:154)
                   at android.app.ActivityThread.main(ActivityThread.java:6077)
                   at java.lang.reflect.Method.invoke(Native Method)
                   at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:866)
                   at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:756)
                Caused by: java.lang.IllegalStateException: Required view 'navigation_view' with ID 2131689604 for field 'navigationView' was not found. If this view is optional add '@Nullable' (fields) or '@Optional' (methods) annotation.
                   at butterknife.internal.Utils.findRequiredView(Utils.java:138)
                   at butterknife.internal.Utils.findRequiredViewAsType(Utils.java:150)
                   at fr.free.nrw.commons.theme.NavigationBaseActivity_ViewBinding.<init>(NavigationBaseActivity_ViewBinding.java:24)
                   at java.lang.reflect.Constructor.newInstance0(Native Method)
                   at java.lang.reflect.Constructor.newInstance(Constructor.java:430)
                   at butterknife.ButterKnife.createBinding(ButterKnife.java:199)
                   at butterknife.ButterKnife.bind(ButterKnife.java:124)
                   at fr.free.nrw.commons.upload.ShareActivity.onCreate(ShareActivity.java:236)
                   at android.app.Activity.performCreate(Activity.java:6662)
                   at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1118)
                   at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2599)
                   at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:2707) 
                   at android.app.ActivityThread.-wrap12(ActivityThread.java) 
                   at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1460) 
                   at android.os.Handler.dispatchMessage(Handler.java:102) 
                   at android.os.Looper.loop(Looper.java:154) 
                   at android.app.ActivityThread.main(ActivityThread.java:6077) 
                   at java.lang.reflect.Method.invoke(Native Method) 
                   at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:866) 
                   at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:756) 
```

--- a/app/src/main/res/layout/activity_multiple_uploads.xml
+++ b/app/src/main/res/layout/activity_multiple_uploads.xml
@@ -1,5 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
 <android.support.v4.widget.DrawerLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
     android:id="@+id/drawer_layout"
     android:layout_width="match_parent"
     android:layout_height="match_parent">
@@ -22,17 +23,12 @@
             android:orientation="vertical"></FrameLayout>
     </RelativeLayout>
 
-    <RelativeLayout
-        android:id="@+id/drawer_pane"
-        android:layout_width="320dp"
-    android:layout_height="match_parent"
+    <android.support.design.widget.NavigationView
+        android:id="@+id/navigation_view"
+        android:layout_width="wrap_content"
+        android:layout_height="match_parent"
         android:layout_gravity="start"
-        android:background="@android:color/white">
-
-        <FrameLayout
-            android:id="@+id/drawer_fragment"
-            android:layout_width="match_parent"
-            android:layout_height="match_parent" />
-    </RelativeLayout>
+        app:headerLayout="@layout/drawer_header"
+        app:menu="@menu/drawer" />
 
 </android.support.v4.widget.DrawerLayout>

--- a/app/src/main/res/layout/activity_share.xml
+++ b/app/src/main/res/layout/activity_share.xml
@@ -36,17 +36,12 @@
         </FrameLayout>
     </RelativeLayout>
 
-    <RelativeLayout
-        android:id="@+id/drawer_pane"
-        android:layout_width="320dp"
+    <android.support.design.widget.NavigationView
+        android:id="@+id/navigation_view"
+        android:layout_width="wrap_content"
         android:layout_height="match_parent"
         android:layout_gravity="start"
-        android:background="@android:color/white">
-
-        <FrameLayout
-            android:id="@+id/drawer_fragment"
-            android:layout_width="match_parent"
-            android:layout_height="match_parent" />
-    </RelativeLayout>
+        app:headerLayout="@layout/drawer_header"
+        app:menu="@menu/drawer" />
 
 </android.support.v4.widget.DrawerLayout>

